src/sinopiaServer.js

```diff
@@ -65,24 +66,37 @@ export const getResourceTemplate = (templateId) => {
     return emptyTemplate
   }
 
+  // TODO: Replace the current return value with a promise, so both functions
+  // called by getResourceTemplate() have a uniform interface, like so:
+  //
+  // return new Promise(resolve => {
+  //   resolve(resourceTemplateId2Json.find((template) => {
+  //     return template.id == templateId
+  //   }).json)
+  // })
   return resourceTemplateId2Json.find((template) => {
     return template.id == templateId
   }).json
 }
 
-export const getResourceTemplateFromServer = async (group, id) => {
-  const emptyTemplate = { propertyTemplates : [{}] }
+const getResourceTemplateFromServer = (templateId, group) => {
+  // Allow function to be called without second arg
```

Comment:
If clients do not specify a group, use the default group. This also helps us retain the prior function signature.